server/cli/launch.py

```diff
@@ -13,22 +14,53 @@
 from server.app.util.utils import custom_format_warning
 from server.utils.utils import find_available_port, is_port_available
 
-
 # anything bigger than this will generate a special message
-BIG_FILE_SIZE_THRESHOLD = 100 * 2**20  # 100MB
+BIG_FILE_SIZE_THRESHOLD = 100 * 2 ** 20  # 100MB
+
+
+def common_args(func):
```

Comment:
function name doesn't mean much to me -- "common" to what?   Maybe just add a comment?

Reply:
common to gui/cli. will add comment